apb.yml

```diff
@@ -34,6 +34,7 @@ plans:
         name: vm_name
         type: string
         required: true
+        pattern: "^[a-z0-9]*$"
```

Comment:
Should we also allow - and . which are commonly used separators?

Reply:
I think it makes sense to add them too